packages/components/stories/contract-card/stories/multiplier-open.js

```diff
@@ -0,0 +1,68 @@
+import React, { useEffect, useState } from 'react';
+import Wrapper from '../../shared/wrapper';
+import ContractCard from 'Components/contract-card';
+import { getCardLabels, getContractTypeDisplay } from '../statics/contract';
+import sampleData from '../sample-data/sampleData.json';
+import '../contract-card.stories.scss';
+
+const Basic = () => {
+    const [loaded, setLoaded] = useState(false);
```

Comment:
Same here. Since these are stories and not real production code, it's not necessary to update here. Kindly remember suggested changes while you're working on our code base.